src/ui/FormWidgets/DatePicker.ts

```diff
@@ -84,7 +86,14 @@ export class DatePicker implements IFormWidget, IFormWidgetSettable {
     this.element.readOnly = true;
     this.picker = new Pikaday({
       field: this.element,
-      onSelect: this.onChange
+      onSelect: this.onChange,
+      i18n: {
```

Comment:
You are adding localization to DatePicker, but you are using i18n (internationalization) instead of i10n (localization).